tests/webview/vendorbooks/test_vendorbooks_load.py

```diff
@@ -0,0 +1,30 @@
+import requests
+from tests import markers
+from pages.webview.home import Home
+
+
+@markers.vendor
+@markers.test_case("C593141")
+@markers.nondestructive
+def test_vendor_books_not_redirecting(vendor_base_url, selenium):
+
+    home = Home(selenium, vendor_base_url).open()
+    books = []
+
+    # GIVEN the openstax list
+    for op_book in home.featured_books.openstax_list:
+
+        books.append(op_book.title)
+
+    for book_title in books:
+
+        home = Home(selenium, vendor_base_url).open()
+
+        title_link = home.driver.find_element_by_link_text(book_title)
```

Comment:
There is selenium driver code in the test... is there a reason this couldn't be in the page object?

Reply:
Similar reason as above. Stale element exception. I tried to replace it with book_cover_link function but getting stale element... very frustrating exception.